CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Help/ViewControllers/HelpTipsViewController.swift

```diff
@@ -75,6 +80,11 @@ final class HelpTipsViewController: UIViewController, HelpBottomBarEnabledViewCo
                 nibName: "HelpTipCell",
                 bundle: giniCaptureBundle()),
             forCellReuseIdentifier: HelpTipCell.reuseIdentifier)
+        tableView.register(
```

Comment:
[Request] We don't need to show the "Useful tips" section header here. The designs were outdated... Sorry for the misunderstanding.

The "Useful tips" section header is only needed on the no results screen.